docs/variants/qemu_q35/development.md

```diff
@@ -0,0 +1,138 @@
+# Dasharo (UEFI) v0.1.0 for QEMU Q35
+
+## Intro
+
+1. Please check your qemu setup and launch qemu outside
+docker instance (host machine) using built OVMF firmware image.
+
++ QEMU Installation:
+    - [Install QEMU on your linux distro](<https://www.qemu.org/download/#linux>)
+
+ ```bash
+ qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd -debugcon file:debug.log -global isa-debugcon.iobase=0x402 -global ICH9-LPC.disable_s3=1 -net none -machine q35,smm=on
```

Comment:
```suggestion
 qemu-system-x86_64 -drive if=pflash,format=raw,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd \
      -debugcon file:debug.log -global isa-debugcon.iobase=0x402 -global ICH9-LPC.disable_s3=1 \
      -net none -machine q35,smm=on
```

Reply:
I believe there should be two FD files one for variables second for the code.

Reply:
Of course otherwise variables are volatile. Please check OST2 Arch4021 2023_v1

Reply:
```suggestion
 qemu-system-x86_64 -machine q35,smm=on \
-global driver=cfi.pflash01,property=secure,value=on \
-drive if=pflash,format=raw,unit=0,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF_CODE.fd,readonly=on \
-drive if=pflash,format=raw,unit=1,file=Build/OvmfX64/DEBUG_GCC5/FV/OVMF_VARS.fd \
-debugcon file:debug.log -global isa-debugcon.iobase=0x402 \
-global ICH9-LPC.disable_s3=
```

This should be the correct command. Also removed the `-net none` as we will want to use network boot as well.

Reply:
@maheshtammisetti you haven't applied it yet

Reply:
Applied the fix